coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
If we want a different format for exceptions we should use a custom formatter IMO. See  the ``Formatter.logException`` method in https://docs.python.org/3/library/logging.html#formatter-objects

@userzimmermann WDYT?